src/Cli/dotnet/Installer/Windows/InstallRequestMessage.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable disable
+
+using System.Text;
+using Microsoft.DotNet.Workloads.Workload.Install.InstallRecord;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Installer.Windows
+{
+    /// <summary>
+    /// Defines the IPC message structure used to send commands from the unelevated client to the elevated server instance of
+    /// the installer
+    /// </summary>
+    internal class InstallRequestMessage : InstallMessageBase
+    {
+        /// <summary>
+        /// The dependent value to modify.
+        /// </summary>
+        public string Dependent
+        {
+            get;
+            set;
+        }
+
+        public string ManifestPath
+        {
+            get;
+            set;
+        }
+
+        public string LogFile
+        {
+            get;
+            set;
+        }
+
+        public string PackageId
+        {
+            get;
+            set;
+        }
+
+        public string PackagePath
+        {
+            get;
+            set;
+        }
+
+        public string PackageVersion
+        {
+            get;
+            set;
+        }
+
+        public string ProductCode
+        {
+            get;
+            set;
+        }
+
+        public string ProviderKeyName
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The type of the install message.
+        /// </summary>
+        public InstallRequestType RequestType
+        {
+            get;
+            set;
+        }
+
+        public string SdkFeatureBand
+        {
+            get;
```

Comment:
Not all message include the feature band, I had lots of issues with null references because the backing field of the struct couldn't be initialized during deserialization. And because structs are value types, they're not null, but the private field that held the ReleaseVersion was null.

Reply:
Gotcha, no big deal, no need then